src/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.cs

```diff
@@ -277,7 +280,10 @@ private void InitManifestModule()
         /// to have a strong name and a hash will be computed when the assembly
         /// is saved.
         /// </summary>
-        [System.Security.DynamicSecurityMethod] // Methods containing StackCrawlMark local var has to be marked DynamicSecurityMethod
+        /// <remarks>
+        /// Methods containing StackCrawlMark local var has to be marked DynamicSecurityMethod.
```

Comment:
This comment does not belong to the public description of the method contract. It belongs next to the `DynamicSecurityMethod` attribute like where it was.

Reply:
oh ok, let me fix that.